<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>_windows_</class>
 <widget class="QMainWindow" name="_windows_">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>473</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>681</width>
    <height>473</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>681</width>
    <height>473</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>_windows_</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(44, 44, 44);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignmentFlag::AlignLeft">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::Shape::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_add">
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>36</width>
           <height>36</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_remover">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="styleSheet">
       <string notr="true">QTableWidget {
background-color: rgb(44, 44, 44);
}
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::NoFrame</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SelectionMode::MultiSelection</enum>
      </property>
      <property name="gridStyle">
       <enum>Qt::PenStyle::NoPen</enum>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Nome</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>N1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>N2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Media</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Resultado</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Falta Media</string>
       </property>
      </column>
     </widget>
    </item>
    <item alignment="Qt::AlignmentFlag::AlignLeft">
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::Shape::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>AI</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="styleSheet">
          <string notr="true">border-radius : 40px;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
